ui/src/views/Quickstart/index.jsx

```diff
@@ -205,7 +205,10 @@ const cmdDirectory = (type, org = '<YOUR_ORG>', repo = '<YOUR_REPO>') =>
     marginBottom: 2 * theme.spacing.unit,
   },
   listStyle: {
-    width: '100%',
+    width: '97%',
+    [theme.breakpoints.down('sm')]: {
+      width: '90%',
+    },
```

Comment:
This makes sure that the width of the text field for ```Repository Name``` doesn't change. Does this look okay? @helfi92